src/node_options.cc

```diff
@@ -50,11 +50,13 @@ void DebugOptions::CheckOptions(std::vector<std::string>* errors,
                       "`node --inspect-brk` instead.");
   }
 
-  std::vector<std::string> destinations =
-      SplitString(inspect_publish_uid_string, ',');
+  std::vector<std::string_view> destinations =
+      SplitString(std::string_view(inspect_publish_uid_string.data(),
```

Comment:
I think that inspect_publish_uid_string is an std::string. Normally, you do have automatic conversion from std::string to std::string_view. So there should be no need for the explicit conversion to std::string_view here.